### PR TITLE
doc: Add known issues for crypto configurations

### PR DESCRIPTION
**Pull request by Vge0rge**
Adds two known issues for some crypto configurations which may cause build failures.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -2594,6 +2594,20 @@ Crypto
...
+
+NCSDK-22091: Selecting :kconfig:option:`NORDIC_SECURITY_BACKEND` and :kconfig:option:`PSA_CORE_BUILTIN` causes build failure
+  Selecting both :kconfig:option:`NORDIC_SECURITY_BACKEND` and :kconfig:option:`PSA_CORE_BUILTIN`  will result in a build failure due to undefined references to different structs.
+
+  **Workaround:** Manually define PSA_CORE_BUILTIN in the file nrf_security/configs/legacy_crypto_config.h.template.
+
+.. rst-class:: v2-4-0 v2-4-1
+
+NCSDK-22593: Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` causes build failure
+  Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` will result in a build failure due to unsatissified dependencies in check_config.h.
```
Comment (SebastianBoe) [inline doc/nrf/releases_and_maturity/known_issues.rst:2607]:
```suggestion
  Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` will result in a build failure due to unsatisfied dependencies in check_config.h.
```

**Comment by b-gent** [comment]
@Vge0rge  ok if I amend with small edits?

**Comment by Vge0rge** [comment]
> @Vge0rge ok if I amend with small edits?

Sure!

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -2594,6 +2594,20 @@ Crypto
...
 The issues in this section are related to :ref:`nrfxlib:crypto`.
 
+.. rst-class:: v2-4-1 v2-4-0
+
+NCSDK-22091: Selecting both :kconfig:option:`NORDIC_SECURITY_BACKEND` and :kconfig:option:`PSA_CORE_BUILTIN` causes a build failure
+  Selecting both :kconfig:option:`NORDIC_SECURITY_BACKEND` and :kconfig:option:`PSA_CORE_BUILTIN` results in a build failure due to undefined references to different structs.
+
+  **Workaround:** Manually define ``PSA_CORE_BUILTIN`` in the file :file:`nrf_security/configs/legacy_crypto_config.h.template`.
+
+.. rst-class:: v2-4-0 v2-4-1
```
Comment (b-gent) [inline doc/nrf/releases_and_maturity/known_issues.rst:2604]:
```suggestion
.. rst-class:: v2-4-1 v2-4-0
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -2594,6 +2594,20 @@ Crypto
...
+
+NCSDK-22593: Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` causes build failure
```
Comment (b-gent) [inline doc/nrf/releases_and_maturity/known_issues.rst:2606]:
```suggestion
NCSDK-22593: Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` causes a build failure
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -2594,6 +2594,20 @@ Crypto
...
+  Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` will result in a build failure due to unsatissified dependencies in check_config.h.
```
Comment (b-gent) [inline doc/nrf/releases_and_maturity/known_issues.rst:2607]:
```suggestion
  Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` results in a build failure due to unsatisfied dependencies in :file:`check_config.h`.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 5]
```diff
@@ -2594,6 +2594,20 @@ Crypto
...
+
+  **Workaround:** Manually define MBEDTLS_AES_C in the file nrf_security/configs/nrf-config.h or enable :kconfig:option:`PSA_WANT_ALG_GCM`.
```
Comment (b-gent) [inline doc/nrf/releases_and_maturity/known_issues.rst:2609]:
```suggestion
  **Workaround:** Manually define ``MBEDTLS_AES_C`` in the file :file:`nrf_security/configs/nrf-config.h` or enable :kconfig:option:`PSA_WANT_ALG_GCM`.
```

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 4 above)
Comment (Vge0rge) [reply doc/nrf/releases_and_maturity/known_issues.rst]: I did it as well but I could commit the suggestion as is because of conflicts. Please check :)

**Review by mswarowsky — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**